### PR TITLE
Refactor : GF Styler 

### PR DESCRIPTION
**Pull request by supriyajjadhav**
### Description
Trello task - https://trello.com/c/mSbMJ7X5/144-uag-block-wise-cleanup

### Types of changes
Breaking change

### Checklist:
- [x] My code is tested
- [x] I've included any necessary tests 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/gf-styler/styling.js [hunk 1]
```diff
@@ -127,546 +127,824 @@ function styling( props ) {
...
-			"line-height" : generateCSSUnit( radioCheckLineHeight, radioCheckLineHeightType ),
-			"color"       : radioCheckLableColor,
-		},
-		" .uagb-gf-styler__check-style-enabled form .gfield_checkbox label" : {
-			"font-size"   : generateCSSUnit( radioCheckFontSize, radioCheckFontSizeType ),
-			"font-family" : radioCheckFontFamily,
-			"font-weight" : radioCheckFontWeight,
-			"line-height" : generateCSSUnit( radioCheckLineHeight, radioCheckLineHeightType ),
-			"color"       : radioCheckLableColor,
+		" .uagb-gf-styler__check-style-enabled .gfield_checkbox input[type='checkbox'] + label:before": {
```
Comment (rohitp-bsf) [inline src/blocks/gf-styler/styling.js:484]: This selector is being used twice, check if we can merge that!

File: src/blocks/gf-styler/styling.js [hunk 2]
```diff
@@ -127,546 +127,824 @@ function styling( props ) {
...
+			'font-size': 'calc( ' + radioCheckSize + 'px / 1.2 )',
+			'border-color': radioCheckBorderColor,
+			'border-style': 'solid',
+			'border-width': generateCSSUnit( radioCheckBorderWidth, 'px' ),
+			'border-radius': generateCSSUnit(
+				radioCheckBorderRadius,
+				radioCheckBorderRadiusType
+			),
+		},
+		" .uagb-gf-styler__check-style-enabled .gfield_checkbox input[type='checkbox']:checked + label:before": {
```
Comment (rohitp-bsf) [inline src/blocks/gf-styler/styling.js:498]: This selector is being used twice, check if we can merge that!

File: src/blocks/gf-styler/styling.js [hunk 3]
```diff
@@ -127,546 +127,824 @@ function styling( props ) {
...
+			'border-color': fieldBorderFocusColor,
+		},
+		" .uagb-gf-styler__check-style-enabled input[type='checkbox'] + label:before": {
```
Comment (rohitp-bsf) [inline src/blocks/gf-styler/styling.js:501]: This selector is being used twice, check if we can merge that!

File: src/blocks/gf-styler/styling.js [hunk 4]
```diff
@@ -127,546 +127,824 @@ function styling( props ) {
...
+			width: generateCSSUnit( radioCheckSize, 'px' ),
+			'font-size': 'calc( ' + radioCheckSize + 'px / 1.2 )',
+			'border-color': radioCheckBorderColor,
+			'border-width': generateCSSUnit( radioCheckBorderWidth, 'px' ),
+			'border-radius': generateCSSUnit(
+				radioCheckBorderRadius,
+				radioCheckBorderRadiusType
+			),
+		},
+		" .uagb-gf-styler__check-style-enabled input[type='checkbox']:checked + label:before": {
```
Comment (rohitp-bsf) [inline src/blocks/gf-styler/styling.js:514]: This selector is being used twice, check if we can merge that!

File: src/blocks/gf-styler/styling.js [hunk 5]
```diff
@@ -127,546 +127,824 @@ function styling( props ) {
...
+			'border-color': fieldBorderFocusColor,
+		},
+		" .uagb-gf-styler__check-style-enabled .gfield_radio input[type='radio'] + label:before": {
```
Comment (rohitp-bsf) [inline src/blocks/gf-styler/styling.js:517]: This selector is being used twice, check if we can merge that!

**Review by rohitp-bsf — CHANGES_REQUESTED** [review]
- Style.scss file having line 418: ` padding: 0px 0px 0px 0px;` can we merge this to ` padding: 0`

**Review by rohitp-bsf — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**